### PR TITLE
Ai blue

### PR DESCRIPTION
**Pull request by BlueNoBaka**
补了几个漏翻的短信、电话
应该还有一些小改动

### CODE REVIEW

**Comment by RemLawrence** [comment]
我打算周末看一下

File: text/MAIN.txt [hunk 1]
```diff
@@ -23856,28 +23856,28 @@
...
-0x2A39A96C=Yo, you didn't show at the hospital. Sumthin happen? Want me to give someone a beatin?
+0x2A39A96C=哟，你没如约来医院。出事了吗？需要我去教训一下谁吗？
 
 ;0xC094D624=Where were u, son. I got enuff to be worried about bein sick without thinking u might be in truble.
-0xC094D624=Where were u, son. I got enuff to be worried about bein sick without thinking u might be in truble.
+0xC094D624=你人呢，小子？光是自己的事我就够头疼了，更别说还要担心你有没有出事了。
 
 ;0xCEE772C9=I done got confused and waited at hospital for u. Damn, Dwaye can be stupid sometimes.
-0xCEE772C9=I done got confused and waited at hospital for u. Damn, Dwaye can be stupid sometimes.
+0xCEE772C9=我一直在医院等你，还很疑惑。妈的，Dwayne偶尔是会犯蠢。
```
Comment (RemLawrence) [inline text/MAIN.txt:23874]: 这句话是Dwayne自己说的吗？

File: text/MF4AUD.txt [hunk 2]
```diff
@@ -300,7 +300,7 @@
 0x745CEE2A=MF3_B_PG
 
 ;0x7792B023=~z~The 'roid rage has screwed up his head so I think I'll just be hangin' with him. No more work.
-0x7792B023=~z~嗑药带来的暴躁应该已经毁了他的脑子，所以我想我最多跟他出去玩玩。不会替他办事了。
+0x7792B023=~z~嗑药带来的暴躁已经毁了他的脑子，所以我想我最多跟他出去玩玩。不会替他办事了。
```
Comment (RemLawrence) [inline text/MF4AUD.txt:303]: 暴躁后面加个名词

File: text/MF5AUD.txt [hunk 3]
```diff
@@ -294,7 +294,7 @@
 0x4A32FD6E=MF3_C_FJ
 
 ;0x4A41B7E8=~z~Roman, you'd never believe it but I met this girl on Craplist.
-0x4A41B7E8=~z~Roman, you'd never believe it but I met this girl on Craplist.
+0x4A41B7E8=~z~Roman，你肯定不敢相信，但我在Craplist约到妹子了。
```
Comment (RemLawrence) [inline text/MF5AUD.txt:297]: this girl有指向性吧？**这妹子**？

File: text/MF5AUD.txt [hunk 4]
```diff
@@ -435,10 +435,10 @@
 0x6F6A9799=MF3_C_ZE
 
 ;0x7014E548=~z~Great, Niko. I've got my hands full at the moment with these charming money lenders I know.
-0x7014E548=~z~真好，Niko。现在我和那些友善的借款人们正忙着呢。
+0x7014E548=~z~真好，Niko。现在我整忙着对付那些友善的债主呢。
```
Comment (RemLawrence) [inline text/MF5AUD.txt:438]: typo

File: text/MAIN.txt (hunk 1 above)
Comment (BlueNoBaka) [reply text/MAIN.txt:23874]: 是的 Dwayne说的

File: text/MF5AUD.txt (hunk 3 above)
Comment (BlueNoBaka) [reply text/MF5AUD.txt:297]: N: Roman, you'd never believe it but I met this girl on Craplist.
R: Who is the lucky lady?
N: Some trust-funder called Alex. I don't really like her but it's something to do, isn't it?
指Alex 但这句强调的主要是**在craplist**居然能约到妹子

**Pull request merged**